libs/sdk-common/src/lib.rs

```diff
@@ -18,6 +18,7 @@ mod utils;
 // use a different version, the Into traits defined here would not be usable by them
 // (e.g. impl From<bip32::Error> for LnUrlError)
 pub use bitcoin;
+pub use hickory_resolver;
```

Comment:
I don't think we need this. This would mainly be useful if callers needed to handle `hickory_resolver`-specific constructs, which isn't the case.